src/sync/push.rs

```diff
@@ -195,9 +194,19 @@ pub async fn push(
 #[cfg(not(target_arch = "wasm32"))]
```

Comment:
same comment here re arch. only the test that uses the network needs this target (test_push). should probably rename also to avoid collision something like test_try_push_http or similar.